app/views/business_mailer/cancel_subscription.text.erb

```diff
@@ -0,0 +1,9 @@
+Hey <%= @business.first_name %>,
+
+Your subscription to RailsDevs is cancelled. Your access to the site will be available until the end of your billing period.
+
+Were you able to hire someone? <%= link_to "Fill out this form", business_hiring_form_url %> and I’ll send you info on next steps.
+
+A friendly reminder that according to the signed <%= link_to "hiring agreement", hiring_agreement_terms_url %> you must notify RailsDevs that you have hired someone no later than the developer's first day of employment.
```

Comment:
We can't use HTML in plain text emails. Please update this to put the link at the end preceded with a colon. You can model it after `app/views/business_mailer/welcome.text.erb`.

Reply:
Sorry did not realized while copying content from html file. Have fixed it.